### ISSUE TITLE
Even though a local branch is located at my windows desktop, it throws a "Can't find ..." prompt

### ISSUE THREAD

**Issue by Ishita-Pandey** [desktop/desktop#10482]
![image](https://user-images.githubusercontent.com/49427683/91844912-e6755780-ec75-11ea-8c65-00c22b1c7bf3.png)

**Comment by steveward** [comment]
Thanks for the report @Ishita-Pandey. To confirm the issue -- when you click `Locate` and navigate to the repository on your Desktop it is not being added correctly? Could you try dragging-and-dropping it directly into GitHub Desktop again to see if that succeeds?

**Comment by Ishita-Pandey** [comment]
Thanks for responding Steve.
I tried both clicking Locate as well as dragging-and-dropping, in both cases the repo shows up momentarily for a second and then goes back to the same "Can't find" prompt

**Comment by steveward** [comment]
@Ishita-Pandey thanks for the update. Is this behavior reproducible with any other repositories? If you move the repository to a different location do you encounter the same behavior?

**Comment by Ishita-Pandey** [comment]
Yes @steveward. This is consistent throughout

**Comment by niik** [comment]
We've also seen this occur due to malformed Git configs or lock files. GitHub Desktop assumes that a repository doesn't exist if it's unable to execute `git status` in the working directory. It would be very helpful if you could upload a log file from GitHub Desktop. To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs in Explorer`.

The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue. Thanks!

**Comment by solocommand** [comment]
@niik I ran into the same issue after cutting+pasting a repository to a new location and attempting to relocate to it in the desktop app. Based on the logs it looks like it had registered the previous location and was continuing to try and access it (via the periodic fetch), but didn't register the new location when locating the repository.

[2020-09-06.desktop.production.log](https://github.com/desktop/desktop/files/5179958/2020-09-06.desktop.production.log)

@Ishita-Pandey I was able to bypass it by removing the local repo and cloning it again to its new home on my disk.
